templates/default.mustache

```diff
@@ -378,6 +378,9 @@ call <sid>hi("StartifySelect",   s:gui0C, "", s:cterm0C, "", "", "")
 call <sid>hi("StartifySlash",    s:gui03, "", s:cterm03, "", "", "")
 call <sid>hi("StartifySpecial",  s:gui03, "", s:cterm03, "", "", "")
 
+" Java Operator color
```

Comment:
Great but this line should read `" Java highlighting` in keeping with the other headings.